src/otx/algo/detection/heads/base_head.py

```diff
@@ -466,12 +424,14 @@ def export_by_feat(
         bbox_preds: list[Tensor],
         score_factors: list[Tensor] | None = None,
         batch_img_metas: list[dict] | None = None,
-        cfg: DictConfig | None = None,
+        cfg: dict | None = None,
```

Comment:
Is this something that can't be removed? Or how about renaming it something else, because the name `cfg` doesn't really make sense to me.

Reply:
I think we should make it more explicit, not to hold everything in "cfg" as it was in mmdet. Now we can make all required arguments here and eliminate any "cfg"

Reply:
I agree with your opinion, but if doing it, the number of arguments will be so many.
Let's talk about this for the next topic.